roles/tas_single_node/tasks/main.yml

```diff
@@ -9,6 +9,28 @@
   ansible.builtin.include_tasks: os.yml
   when: not tas_single_node_skip_os_install
 
+- name: Create cockpit-user
```

Comment:
I might be missing something, but why do we need the user?

Reply:
So when working on this pr, I found that I was unable to log into the cockpit console with the cloud-user (until I set a password) as there didn't seem to have a password set, I figured it also could be the case for other rhel machines so I thought it best to include it.